services/api/emails/layout.html

```diff
@@ -314,14 +313,13 @@
                   width="570"
                   cellpadding="0"
                   cellspacing="0"
-                  role="presentation"
-                >
+                  role="presentation">
                   <tr>
                     <td class="content-cell" align="center">
-                      <p class="f-fallback sub align-center">&copy; 2021 {{appName}}. All rights reserved.</p>
+                      <p class="f-fallback sub align-center">&copy; 2021 {{APP_NAME}}. All rights reserved.</p>
```

Comment:
lets generate the year (2021) so we don't have to update this every year

Reply:
ah yeah good idea... was thinking about that too